components/massmint/EditPanel.vue

```diff
@@ -100,6 +107,7 @@ import {
   NeoSidebar,
 } from '@kodadot1/brick'
 import type { NFT } from './types'
+import CustomAttributeInput from '@/components/rmrk/Create/CustomAttributeInput.vue'
```

Comment:
we should slowly get rid off the `rmrk` folder

Reply:
done https://github.com/kodadot/nft-gallery/pull/11267/commits/71bbd1b82dcb50c4d443b7e537168faad8a8015e